### PR TITLE
[wip] Add more api tests

### PR DESCRIPTION
**Pull request by paliwodar**

### CODE REVIEW

File: cypress/integration/backend.spec.js [hunk 1]
```diff
@@ -113,6 +113,74 @@ describe('Transaction Management Backend - Level 1', () => {
     })
   })
 
+  t('Can read all created transactions', () => {
```
Comment (fenske) [inline cypress/integration/backend.spec.js:116]:
```suggestion
  it('Can read all created transactions', () => {
```

**Review by fenske — CHANGES_REQUESTED** [review]
Thanks for the PR!

The best way to validate test suite changes would be running them against a reference implementation.

In this case, we have https://infra.devskills.app/api/transaction-management that's referred from inside [the OpenAPI spec](https://infra.devskills.app/transaction-management/api/3.1.0) (here's [the implementation](https://github.com/devskills-robot/infra/tree/master/pages/api/transaction-management) itself).

I ran both on the pipeline and Gitpod ([link](https://gitpod.io/#https://github.com/DevSkillsHQ/transaction-management-backend-level-1)) and it failed on the new test case. See Cypress images below:

Transaction Management Backend - Level 1 -- Can handle invalid requests (failed)

Transaction Management Backend - Level 1 -- Can read all created transactions (failed)

Did it work for you when you tested it?

File: cypress/integration/backend.spec.js (hunk 1 above)
Comment (paliwodar) [reply cypress/integration/backend.spec.js]: Fixed!

**Comment by paliwodar** [comment]
> I tested it on a separate branch and all tests pass now. I saw you mentioned it in Notion that you'd think about adding more tests. Do you want to do it as part of this PR or separate?

Fine to merge this one for me. I'd add more if any in a separate one.

**Comment by paliwodar** [comment]
> I saw you mentioned it in Notion that you'd think about adding more tests. Do you want to do it as part of this PR or separate?

IMO we are good with the tests. I will raise this in Notion.

**Pull request merged**